getting_started/introduction/introduction_to_godot.rst

```diff
@@ -0,0 +1,126 @@
+Introduction to Godot
+=====================
+
+This article is here to help you figure out whether Godot might be a good fit
+for you. We will introduce some broad features of the engine to give you a feel
+for what you can achieve with it and answer questions such as "what do I need to
+know to get started?".
+
+This is by no means an exhaustive overview. We will introduce many more features
+in this getting started series.
+
+What is Godot?
+--------------
+
+Godot is a general-purpose 2D and 3D game engine designed to support all sorts
+of projects. You can use it to create games or applications you can then release
+on desktop or mobile, as well as on the web.
+
+You can also create console games with it, although you either need strong
+programming skills or a developer to port the game for you.
+
+.. note:: The Godot team can't provide an open-source console export due to the
+          licensing terms imposed by console manufacturers. Regardless of the
+          engine you use, though, releasing games on consoles is always a lot of
+          work. You can read more on that here:
```

Comment:
Missing link 🙃

Reply:
Could you review the intro in its PR? It's #4302. This PR is just based on it because there are links to the intro, but 4302 should be reviewed and merged first.

Reply:
Oh, this is confusing...

Reply:
I updated the first message to reflect that. I just split the work into two PRs hoping the other would get reviews and merged while this one was a draft.